pkg/pillar/cas/cas.go

```diff
@@ -12,28 +12,29 @@ import (
 
 // BlobInfo holds the info of a blob present in CAS's blob store
 type BlobInfo struct {
-	//Digest to identify the blob uniquely. The format will/should be <algo>:<hash> (currently supporting only sha256:<hash>).
+	// Digest to identify the blob uniquely. The format will/should be <algo>:<hash> (currently supporting only sha256:<hash>).
```

Comment:
If you fix some of these it appears like you need to fix many more in the same functions and structures.

Reply:
On the first run, it flagged about 11 of these in the `cas.go` file. I didn't touch it, so wasn't sure why it complained, but I didn't mind an extra commit to fix it. I might have opened a Pandora's Box.

Reply:
I gave it one more shot to address those. If it gets out of hand, I will remove the commit.

Do you know why it flagged the file, even though I did not touch it?

Reply:
I addressed a large number of them, all in files I did not go near. I think I have hit my limit on that.